tests/sqlite3/Makefile.basic

```diff
@@ -0,0 +1,12 @@
+COMPILER = g++
```

Comment:
These Makefiles are useful on test systems that have minimal set of packages installed.
As to having one, single build system - these Makefiles played this role from the beginning of the project, even before CMake was introduced.